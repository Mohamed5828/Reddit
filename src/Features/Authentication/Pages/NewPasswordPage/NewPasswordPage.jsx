import { PageContainer, Photo, LogInContainer } from "./NewPasswordPage.styled";
import NewPasswordPageCom from "Features/Authentication/Layouts/NewPasswordPageCom/NewPasswordPageCom";

/**
 * NewPasswordPage component (The page of new password)
 * @returns {React.Component}  NewPasswordPage component (The page of new password)
 */
const NewPasswordPage = () => {
  return (
    <PageContainer>
      <Photo></Photo>

      <LogInContainer>
        <NewPasswordPageCom></NewPasswordPageCom>
      </LogInContainer>
    </PageContainer>
  );
};

export default NewPasswordPage;
