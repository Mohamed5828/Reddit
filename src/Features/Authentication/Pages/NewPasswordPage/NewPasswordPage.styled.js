import styled from "styled-components";

export const PageContainer = styled.div`
  display: flex;
  justify-content: center;
  @media (max-width: 330px) {
    align-items: center;
    margin: 50px auto;
  }
`;

export const Photo = styled.div`
  background-image: url(https://www.redditstatic.com/accountmanager/bbb584033aa89e39bad69436c504c9bd.png);
  width: 10%;
  height: 100vh;
  min-height: 430px;
  background-repeat: no-repeat;
  background-size: cover;


  @media (max-width: 330px) {
    display: none;
    padding:20px;
  }
`;

export const LogInContainer = styled.div`
 width: 90%;
 display:flex;
 align-items: center;

 >div {
    margin:0 0 0 20px;

    @media (max-width: 330px) {
      margin: auto;
    }
 }

 @media (max-width: 330px) {
  width: 100%;
  margin: auto;
}
`;



