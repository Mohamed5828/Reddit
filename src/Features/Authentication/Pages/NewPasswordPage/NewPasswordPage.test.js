import { shallow } from "enzyme";

import NewPasswordPage from "./NewPasswordPage";

describe("NewPasswordPage Component", () => {
  it("this is a test for NewPasswordPage Component", () => {
    expect(shallow(<NewPasswordPage />)).toMatchSnapshot();
  });
});
