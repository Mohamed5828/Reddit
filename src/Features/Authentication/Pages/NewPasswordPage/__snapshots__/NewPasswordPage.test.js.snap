// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewPasswordPage Component this is a test for NewPasswordPage Component 1`] = `
<styled.div>
  <styled.div />
  <styled.div>
    <NewPasswordPageCom />
  </styled.div>
</styled.div>
`;
